### ISSUE TITLE
Test dataloader should not set shuffle to True

### ISSUE THREAD

**Issue by charliezon** [joshuacwnewton/d-SNE-PyTorch#3]
https://github.com/joshuacwnewton/d-SNE-PyTorch/blob/68904ce16e5133dcca8a88e0bfb4393eb914847e/dsne_pytorch/data_loading/data_classes.py#L225
